store/store_test.go

```diff
@@ -653,38 +653,75 @@ const MockSearchJSON = `{
 }
 `
 
-func (t *remoteRepoTestSuite) TestUbuntuStoreFind(c *C) {
+func (t *remoteRepoTestSuite) TestUbuntuStoreFindQueries(c *C) {
+	n := 0
 	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
that wouldn't work though, because you're looking for the query passed in to find in the request passed to the store?